stob/ad.c

```diff
@@ -1080,8 +1079,8 @@ static uint32_t stob_ad_write_map_count(struct m0_stob_ad_domain *adom,
 	bool                   eov;
 	struct m0_ivec_cursor  it;
 
-	M0_ENTRY("dom=%p bshift=%u babshift=%d pack=%hhx", adom,
-		 adom->sad_bshift, adom->sad_babshift, pack);
+	M0_ENTRY("dom=%p bshift=%u babshift=%d pack=%x", adom,
```

Comment:
s
/M0_ENTRY("dom=%p bshift=%u babshift=%d pack=%x", adom,
/M0_ENTRY("dom=%p bshift=%u babshift=%d pack=0x%x", adom,

Otherwise this result may be misleading.

Reply:
Will use %#x.